backend/gke/deploy.sh

```diff
@@ -22,6 +22,7 @@ fi
 
 git clone https://github.com/SUSE/cap-terraform.git
 pushd cap-terraform/gke || exit
+git checkout -t origin/prabal-gke-clustername-tfvar
```

Comment:
We shouldn't have this.

Reply:
Nevermind, I saw that the next commit removes it (it would be better to not have it in history for `git bisect`, etc)